projects/aca-content/src/lib/components/search/search-save/list/ui-list/saved-searches-list.ui-component.html

```diff
@@ -0,0 +1,18 @@
+<div class="aca-save-search-list-container">
```

Comment:
instead wrapping this with extra div we can probably apply styles directly to saved-searches-list component tag?

Reply:
Done